Tests/MongoSwiftTests/DNSSeedlistTests.swift

```diff
@@ -85,17 +83,42 @@ final class DNSSeedlistTests: MongoSwiftTestCase {
                 continue
             }
 
-            expect(try doTest()).toNot(throwError(), description: testCase.comment ?? "")
+            let connStr: ConnectionString
+            do {
+                connStr = try doTest()
+            } catch {
+                expect(error).to(beNil(), description: testCase.comment ?? "")
```

Comment:
This expectation will always fail right? It's probably better to just use `XCTFail` or `fail` directly. Also, I think the `continue` can be omitted.

Reply:
I switched to `XCTFail`. however, the `continue` is needed for the compiler to let me use `connStr` below; otherwise, it doesn't know the variable has definitely been defined when I try to use it below.

I considered not wrapping this in a do/catch at all but then we wouldn't get the name of the test case in the failure message.